### PR TITLE
[SEM-036] logo 

### PR DESCRIPTION
**Pull request by angel148**

### CODE REVIEW

**Comment by DanX3N0** [comment]
PR:
image
image
image

FIGMA:
image

it's according to figma.

File: src/ui/components/atoms/logo/style.scss [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    background-color: $ads-color-main;
+  }
+
+  &--primary {
+    background-color: $ads-background-main;
+  }
+
+  &--secondary {
+    background-color: $ads-primary-main;
+  }
```
Comment (Ditmar) [inline src/ui/components/atoms/logo/style.scss:20]: ok, I don't see the media queries referring to the responsive design, there exists a difference between deskop and mobile, review the figma

File: src/ui/components/atoms/logo/style.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
+@import '../../../../globals/variables';
+@import '../../../../globals/mixins';
+
+.logo-image {
+  border-radius: 0 0 1.5rem;
+  height: 51px;
+  padding-top: 0;
+  width: 130px;
```
Comment (Ditmar) [inline src/ui/components/atoms/logo/style.scss:8]: This measure is bab the correct is 130 - 27 = 103 in mobile and ins desktop is other

**Review by Ditmar — CHANGES_REQUESTED** [review]

**Review by Ditmar — CHANGES_REQUESTED** [review]
You are dragging  149 commits and changing 126 the rebase fail, I won't review this pr

**Review by Ditmar — CHANGES_REQUESTED** [review]
bad rebase

**Pull request closed without merging**